app/views/lockout_mailer/lockout_report.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>An educator has been locked out of Student Insights</h1>
+    <p>
+      The following educator has been locked out because they made too many login attempts:
```

Comment:
Small improvement would be to read in the max login Devise config and include it here to make this more precise.